aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -289,6 +299,11 @@ def create_exporter(self):
         ]:
             temporality_dict[typ] = AggregationTemporality.DELTA
 
+        if _is_lambda_environment():
```

Comment:
To work with ADOT Collector, we might prioritize environment variable rather than is_lambda_environment()